package.json

```diff
@@ -27,15 +27,15 @@
     "src/Vitest.res"
   ],
   "peerDependencies": {
-    "rescript": "^10.1.0 || ^11.0.0-alpha.0",
+    "rescript": "^10.1.0 || ^11.0.0",
     "vite": "^2.0.0 || ^3.0.0 || ^4.0.0 || ^5.0.0"
   },
   "dependencies": {
-    "vitest": "^0.34.6"
+    "vitest": "~0.34.6 || ^1.0.0"
```

Comment:
It would be nice to have it separate from the vitest upgrade. Can you exclude this part from the change?

Reply:
Good point, I backed that out

Reply:
can you update lockfile?